docs/content/direct/release-testing.md

```diff
@@ -0,0 +1,31 @@
+# Testing a new KubeStellar Release
+
+The following testing process should be applied to every KubeStellar new release in order to validate the release, this include both regular releases and release candidates. All the tests should be done while the KubeStellar code is still under code-freeze and new code shouldn't be merged into the main branch until all tests are passed and the release is officially declared as ready.  
+In case the release tests fail (even one of them), the release should be declared as instable and a fix through a new release candidate should be worked on ASAP. The KubeStellar code-freeze should be lifted only after all tests are passed and a the release was completed.  
+To reduce the exposure of instable releases the update of the KubeStellar site [kubestellar.io](https://docs.kubestellar.io) should be done only once all release tests passed sussceffuly. 
+
+## Release tests
+The following section describe the tests that must be executed for each release.
+
+Our release tests consists of:
+   * Automatic tests running on Ubuntu X86 (see below)
+   * Manually initiated tests running on OCP (TODO: add specific version and machine details)
+
+Due to the lack of OCP based automatic testing, these tests will be performed only once a release candidate passed all other tests and is a candidate to become a regular release. 
+
+Note:  We plan to automate all release tests in the future
+
+### Automatic (github based) release tests
+KubeStellar CICD automatically runs couple of e2e tests on each new release. Currently these tests include 2 main test types bash based e2e tests and ginkgo based e2e tests. The bash test basically test the scenario of  [multi-cluster workload deployment with kubectl](examples.md#scenario-1---multi-cluster-workload-deployment-with-kubectl). The ginkgo test cover the [Singleton status test](examples.md#scenario-4---singleton-status), and several other tests that are listed in the test [README](https://github.com/kubestellar/kubestellar/blob/main/test/e2e/ginkgo/README.md). Note, however, that the content of the releases tests may be changed in the future. We will refer to those tests as the **e2e release tests**. 
```

Comment:
s/couple/a set of/

Reply:
s/The bash test basically test/The bash test basically tests/

Reply:
fixed